code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -16,7 +16,7 @@
 	return
 
 /obj/item/weapon/gun/projectile/automatic/pistol/m1911
-	name = "\improper M1911 pistol"
+	name = "M1911"

```

Comment:
isn't that applicable to a lot of different guns, though? not just pistols? a lot of them share the same in-hand, the same gunshot sound.

why do you want this to be called m1911 only, while being happy to call the stetchkin a pistol and just name the captain's gun a laser gun?